src/main/java/io/github/bzdgn/merdiven/statement/data/DataStatement.java

```diff
@@ -2,21 +2,84 @@
 
 import io.github.bzdgn.merdiven.statement.BaseStatement;
 import io.github.bzdgn.merdiven.statement.StatementType;
+import io.github.bzdgn.merdiven.statement.data.wrapper.Wrapper;
+import io.github.bzdgn.merdiven.statement.data.wrapper.WrapperFactory;
 
 public class DataStatement extends BaseStatement {
+
+	private static final String DEFAULT_PARAM_WRAPPER = "";
+	private static final String DEFAULT_VALUE_WRAPPER = "\"";
 	
+	private final DataType dataStatementType;
 	private final String paramName;
 	private final String paramValue;
+	private final String operator;
+	private final Wrapper paramWrapper;
+	private final Wrapper valueWrapper;
+
+	public DataStatement(String paramName, String paramValue, String operator, DataType dataStatementType) {
+		this(
+			paramName,
+			paramValue, 
+			operator,
+			getDefaultWrapper(DEFAULT_PARAM_WRAPPER),
+			getDefaultWrapper(DEFAULT_VALUE_WRAPPER),
+			dataStatementType
+		);
+	}
+	
+	public DataStatement(String paramName, String paramValue, String operator, Wrapper wrapper, DataType dataStatementType) {
```

Comment:
Wouldn't it be more consistent and less duplication if this constructor also called the one with two wrappers, just like the previous one?